ocaml/lib/snarky_bindings.ml

```diff
@@ -178,6 +178,15 @@ module Gates = struct
                    { kind = Zero; values = [| in1; in2; out |]; coeffs = [||] }
                 )
           } )
+
+  let basic sl l sr r so o sm sc =
```

Comment:
I assumed that it made sense to expose the basic gate and do all of the configuration in TS instead of exposing `and_equation` as in the [OCaml implementation](https://github.com/MinaProtocol/mina/blob/d5ae586319a9d0fdfd322b2b8b54beb5bfaeddbb/src/lib/crypto/kimchi_backend/gadgets/bitwise.ml#L476). 
1. Is this a correct assumption?
1. Are the argument names good?
1. Is there anything else I should expose while I'm here?